modules/virt-importing-vm-cli.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm-cli_{context}']
+= Importing a Red Hat Virtualization virtual machine with the CLI
+
+You can import a Red Hat Virtualization virtual machine with the CLI by creating the Secret and VirtualMachineImport Custom Resources (CRs). You apply the VirtualMachineImport configuration to start the import process and you can follow its progress.
+
+.Procedure
+
+. Create the Secret CR to store the RHV Manager credentials:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: rhv_credentials
+  namespace: default <1>
+type: Opaque
+stringData:
+  ovirt: |
+    apiUrl: "https://<RHVM_FQDN>:8443/ovirt-engine/api" <2>
+    username: admin@internal
+    password: <3>
+    caCert: |
+      -----BEGIN CERTIFICATE-----
+      <4>
+      -----END CERTIFICATE-----
+----
+<1> Optional. You can specify a different namespace.
+<2> Specify the FQDN of the RHV Manager.
+<3> Specify the password for `admin@internal`.
+<4> Specify the link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html-single/rest_api_guide/index#obtaining_the_ca_certificate[RHV Manager CA certificate].
+
+. Create the VirtualMachineImport CR to configure the virtual machine import process:
++
+[source,yaml]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: VirtualMachineImport
+metadata:
+  name: vm_import
+  namespace: default
+spec:
+  providerCredentialsSecret:
+    name: rhv_credentials
+    namespace: default
+  targetVmName: vm_example
+  startVm: true
+  source:
+    ovirt:
+      vm:
+        id: <80554327-0569-496b-bdeb-fcbbf52b827b> <1>
+      mappings:
+        networkMapping:
```

Comment:
should be: networkMapping**s**

Reply:
I fixed that on the vm-import-operator example as well https://github.com/kubevirt/vm-import-operator/pull/329

Reply:
Fixed